node/README.md

```diff
@@ -109,6 +109,8 @@ file, this will pull the configured tests in and then add them to the test
 workflow.
 
 ```zsh
+$ deno task node:setup
+// or
 $ deno run --allow-read --allow-net --allow-write node/_tools/setup.ts
```

Comment:
I think we should get rid of this because it's something that could get out of date with `deno task node:setup`. People should be using the latest deno if they're contributing.